recipes/preprocess/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "1.2.2" %}
+{% set git_rev = "d5ab9a0544cd2b15b0e8ae4f0b143b9047d1a54b" %}
+
+package:
+  name: preprocess
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/hplgit/preprocess.git
+  git_rev: {{ git_rev }}
```

Comment:
Sorry for not replying earlier. I did report an [issue](https://github.com/hplgit/preprocess/issues/8) for this upstream but never got a reply. I do not think it is likely that we will see any release tarballs or tags. Unfortunately, Hans Petter Langtangen, which maintained `preprocess`, passed away last week after long time illness.